api/proto/openyolo.proto

```diff
@@ -21,9 +21,8 @@ package org.openyolo;
 option java_package = "org.openyolo.proto";
 
 message CredentialRetrieveRequest {
-    repeated string       authDomains        = 1; // at least one required
-    repeated string       authMethods        = 2; // at least one required
-    repeated KeyValuePair additionalParams   = 3;
+    repeated string       authMethods        = 1; // at least one required
```

Comment:
Done. Marked as deprecated and added comment (In javadoc style, not sure what the style guide is proto comments).